scrypt.js

```diff
@@ -2,4 +2,5 @@ document.body.innerHTML = '<h1>hello js</h1>';
 console.log('test1')
 console.log('test2!!!!!!!!!!')
 console.log('test2!!!!!!!!!!')
```

Comment:
Maybe let's remove duplicated console.log?

Reply:
@owner6 what do you think? Do we need the same console log twice?

Reply:
I think let's delete it